backend/src/main/java/com/empire/ArtificialIntelligence.java

```diff
@@ -1021,13 +1021,14 @@ boolean feasible() {
 		FeedPeople() {
 			int turnsUntilHarvest = 4 - world.date % 4;
 			double totalStocks = 0;
-			double totalDemand = 0;
+			double totalDemand = 0.001;  // prevent a divide by zero with no citizens
 			for (Region r : world.regions) {
 				if (!whoami.equals(r.getKingdom())) continue;
 				totalStocks += r.food;
+				if (world.date % 4 == 0) totalStocks += r.crops;
```

Comment:
Hm, this makes me wonder if the computation of turnsUntilHarvest is buggy. (Should it be 3 - world.date % 4?) If that is adjusted, then probably need to check the whole denominator for zero rather than just the totalDemand factor.

Reply:
I believe this is working, with the quirk that 'week-of-harvest' calculates 4, not 0.  Which I think is good here, since harvest week you want to plan distance to the -next- harvest, not just assume you're good.

This is still relatively dumb in not stockpiling ahead of winter, but I think that's fine for AI.